### PR TITLE
Added Tests for login for line coverage

### PR DESCRIPTION
**Pull request by anhuwi**
added test for login of user:
- login success
- login fails
- servicecontrol checkpassword

### CODE REVIEW

File: src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
  * This tests if the UserController works.
  */
 @WebMvcTest(UserController.class)
-public class UserControllerTest {
```
Comment (daveking7) [inline src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java:38]: Why did you change it from public to protected?

**Review by daveking7 — APPROVED** [review]
good

File: src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java (hunk 1 above)
Comment (anhuwi) [reply src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java:38]: a sonarQube thing;)
